hack/builder/Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG ARCH
-FROM ${ARCH}/fedora:32
+FROM quay.io/centos/centos:stream8
```

Comment:
```
ARG ARCH
--platform="linux/${ARCH}"
```

Reply:
@zhlhahaha thanks, I'll try that! Last time I attempted a multi-arch build with Docker tooling it was not quite that simple to get things working, but maybe things have improved significantly in the meantime :)

Reply:
> @zhlhahaha thanks, I'll try that! Last time I attempted a multi-arch build with Docker tooling it was not quite that simple to get things working, but maybe things have improved significantly in the meantime :)

OK, I just verified it with the updated Dockerfile in my x86 server. One more line needs to be added to in the build.sh

Reply:
I ultimately went for a slightly different approach, but your hints were instrumental in getting it to work and so I've credited you accordingly with Suggested-by tags.

I would also appreciate it if you could take a look at my solution and let me know if you spot any flaws in it... You seem to know a lot more than me about multi-arch containers :)